Library/Source/NativeWindow.cpp

```diff
@@ -98,6 +109,14 @@ namespace Babylon
         return Napi::Value::From(info.Env(), decodedData);
     }
 
+    void NativeWindow::AddEventListener(const Napi::CallbackInfo& info)
+    {
+    }
```

Comment:
nit: Log out a warning or something if these methods ever get called? Just thinking it might be confusing down the line if we forget these are stubs.

Reply:
I don't think I can do better than adding a comment here.

Reply:
Can you add a TODO? I'm currently thinking that once everything's plugin-ified, `NativeEngine` should take a dependency on `Console` anyway so it can log stuff out.